### DISCUSSION TITLE
Insert key doesn't work

### DISCUSSION THREAD

**Discussion by Petitjean2** [danielkrupinski/Osiris#3835] [Q&A]
Hello, i'v injected correctly the dll file into csgo.exe, but when i press inser key nothing happen. 

Any idea ?

I'm on CSGO on Steam. Bot Game.

THX

**Comment by Xsintashi** [comment]
use better injector

**Reply by Petitjean2** [reply]
Which one do you recommend ?

I used Remote DLL, and it said "successful".

**Reply by Petitjean2** [reply]
PS : i'm on win 10

**Reply by Xsintashi** [reply]
csGhost v4

**Reply by Petitjean2** [reply]
THX !
